### DISCUSSION TITLE
Ability to add 'component fields' to sections.

### DISCUSSION THREAD

**Discussion by bengreenbank** [craftcms/cms#9075] [Ideas] [answered]
It would be good to be able to group fields into components, and then be able to just add the component to a section, rather than adding each individual field to the section one at a time.

For example, rather than adding:
- Intro Title
- Intro Image
- Intro Link

You can group them into an 'Intro' component and then just add the component to the section. This component could then update in every section it is used each time a new field is added to the component.

**Comment by bengreenbank** [comment]
Potentially a matrix field could actually be a good substitute for the problem I described.

**Comment by brandonkelly** [comment, marked as answer]
Couple related ideas: #8716, #8404
